system/mp/test/mptest.cpp

```diff
@@ -235,7 +241,7 @@ void Test3(IGroup *group,ICommunicator *comm)
         comm->recv(mb,RANK_ALL,MPTAG_TEST,&r);
         StringAttr str;
         mb.read(str);
-        PrintLog("(3) Received '%s' from rank %d",str.get(),r);
+        PrintLog("(3) Received '%s' from rank %u",str.get(),r);
```

Comment:
not that this is wrong, but this looks like a change from elsewhere and a git rebasing type issue..
as does other changes in this file.